.github/workflows/prepare-for-minor-release.yml

```
@@ -18,30 +18,27 @@ jobs:
     timeout-minutes: 60
```

Seems like this whole workflow should always be pretty fast (?)
```suggestion
    timeout-minutes: 10
```